cmd/gb-vendor/update.go

```diff
@@ -119,12 +119,16 @@ Flags:
 				return err
 			}
 
```

Comment:
github won't let me comment on line 73, but please move line 73 down to around 117, the sequence should be

1. fetch dependency into tempdir
2. remove old copy
3. copypath the new copy
4. write back manifest
5. wc.Destroy